update_ip/services/__init__.py

```diff
@@ -1,5 +1,37 @@
-from nfsn import NearlyFreeSpeechService
-from webfaction import WebFactionService
+import inspect
+import os
+import imp
+import glob
+from base import BaseDNSService
 
-services= [WebFactionService, NearlyFreeSpeechService]
```

Comment:
With a good and ready for whatever codebase more people can integrate more services.

Honestly, I was about to write my own program on the first glance, but finally I understood better how the services were created, and I set to work.

It need more comments and documentation, but is a nice tool, let's go to do it better.